### ISSUE TITLE
Can't link to notes homepage

### ISSUE THREAD

**Issue by binyamin** [binyamin/eleventy-garden#28]
## Describe the bug
There's no way to link to the notes homepage (`/notes/`), via wikilinks.

link text|href|backlink?
---|---|---
`notes`|`/notes/notes`|yes
`/` or `/notes`|`/notes/notes`|no
`.` or `./`|n/a|no

## Expected behavior
**In General**: Links including filename, or a relative path _to_ the file, should point to the specified file.
**Here**:  Accept `/` and `./` (relative paths), and `notes` (filename) for the link location.

**Comment by r357** [comment]
Hi, I'm not sure if this helps (I'm not very familiar with js), but I noticed that backlinks don't appear in case of folders inside the notes folders. For example: 

>/notes/Music/index.md
>/notes/Music/Production.md

Linking [[Music/Production.md]] in the index.m creates a clickable link on the Music index site, but the Production site won't show a backlink. However, linking [[Production]] in the index will create a non-clickable link on the Music site, but it will somehow create a backline on the Production site. Also, I'm finding that using [[Music/Production | Production]] and adding 

> [comment]: # ([[Production]])

does both things. Is this just me not doing something right?

**Comment by binyamin** [comment]
@r357 no, you're doing the right thing. This is a bug. Would you mind opening an issue, so we can keep track of this?
